java/rocksjni/jnicallback.cc

```diff
@@ -49,5 +49,20 @@ JniCallback::~JniCallback() {
 
   releaseJniEnv(attached_thread);
 }
+
+void JniCallback::catchAndLog(JNIEnv* env) const {
+  jboolean hasException = env->ExceptionCheck();
+  if (hasException == JNI_TRUE) {
+    env->ExceptionDescribe();
+    env->ExceptionClear();
+  }
+}
+
+void JniCallback::throwJavaLangError(JNIEnv * env, const char * message) const {
+  jclass cls = env->FindClass("java/lang/Error");
```

Comment:
Why `java.lang.Error`?

Have you seen `RocksDBExceptionJni` in `portal.h`?

Reply:
it has been replaced